ksqldb-common/src/main/java/io/confluent/ksql/metrics/MetricCollectors.java

```diff
@@ -143,22 +142,16 @@ public static void addConfigurableReporter(
   }
 
   public static Map<String, Object> addConfluentMetricsContextConfigs(
-      final String ksqlServiceId
+      final String ksqlServiceId,
+      final String kafkaClusterId
   ) {
     final Map<String, Object> updatedProps = new HashMap<>();
     updatedProps.put(RESOURCE_LABEL_TYPE, KSQL_RESOURCE_TYPE);
     updatedProps.put(RESOURCE_LABEL_CLUSTER_ID, ksqlServiceId);
-    return updatedProps;
-  }
-
-  public static Map<String, Object> addConfluentMetricsContextConfigsForKsql(
-      final String ksqlServiceId
-  ) {
-    final Map<String, Object> updatedProps = new HashMap<>();
     updatedProps.put(RESOURCE_LABEL_KSQL_SERVICE_ID, ksqlServiceId);
+    updatedProps.put(RESOURCE_LABEL_KAFKA_CLUSTER_ID, kafkaClusterId);
```

Comment:
if ksql.cluster.id and ksql.service.id are the same, why do we need both?

Reply:
we can remove RESOURCE_LABEL_KSQL_SERVICE_ID and just keep the cluster id label. The service id of a KSQL server is the only id for it so it essentially is the cluster id

Reply:
RESOURCE_LABEL_CLUSTER_ID and RESOURCE_LABEL_KAFKA_CLUSTER_ID are the same. I would recommend to keep RESOURCE_LABEL_KSQL_SERVICE_ID which is more meaningful for KSQL.

Reply:
Oh I see, I was under the impression that the cluster id meant the ksql cluster id, not the kafka one. If that's the case, then I'll keep RESOURCE_LABEL_KSQL_SERVICE_ID

Reply:
@stevenpyzhang @xiaodongdu I think it makes more sense to keep `ksql.cluster.id` as the KSQL service ID, and `ksql.kafka.cluster.id` as the backing cluster id to align with the terminology we already use in RBAC https://docs.confluent.io/current/security/rbac/rbac-get-cluster-ids.html#ksqldb-example

We should fix the other components if they don't align.

Reply:
@xvrl I've updated the PR to match with this ^